Trimble.Modus.Components/Controls/TMSwitch/TMSwitch.xaml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```

Comment:
As expected, iOS has some alignment issue. I will take care of that

<img src="https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/3ab75a82-0588-45b5-80a9-bb1a7bb6c495"  width=20% >

Reply:
This is fixed now